build.gradle

```diff
@@ -337,6 +340,9 @@ project('controller') {
     applicationName = "pravega-controller"
     mainClassName = "io.pravega.controller.server.Main"
     applicationDefaultJvmArgs = ["-server", "-Xms128m", "-XX:+HeapDumpOnOutOfMemoryError",
+                                 "-XX:+PrintGCDetails", "-XX:+PrintGCDateStamps",
+                                 "-Xloggc:PRAVEGA_APP_HOME/logs/gc.log", "-XX:+UseGCLogFileRotation",
```

Comment:
We have have a single docker image for controller and segmentstore. The path of the logs are the same for both of them, inside the container i.e. PRAVEGA_APP_HOME/logs which maps to `/opt/pravega/`.
Also, gc logs are not enabled for pravega singlenode ( The start up script for single node is different from that of controller and SS)